api/task.go

```diff
@@ -152,6 +160,15 @@ type TaskDatabaseDataUpdatePayload struct {
 	BinlogFileEnd   string `json:"binlogFileEnd,omitempty"`
 	BinlogPosStart  int64  `json:"binlogPosStart,omitempty"`
 	BinlogPosEnd    int64  `json:"binlogPosEnd,omitempty"`
+	// RollbackTaskState is a state machine
+	// 0. The initial state is "".
+	// 1. When the rollback generation starts, we set it to "RUNNING".
```

Comment:
What if Bytebase started the generation and crashed immediately?

Reply:
Do you mean that the rollbackTaskState is still ""? Them we should add it to the query condition too.

Reply:
Removed the state.